shell/src/main/java/org/apache/zeppelin/shell/ShellInterpreter.java

```diff
@@ -47,7 +47,10 @@
  */
 public class ShellInterpreter extends KerberosInterpreter {
   private static final Logger LOGGER = LoggerFactory.getLogger(ShellInterpreter.class);
+
   private static final String TIMEOUT_PROPERTY = "shell.command.timeout.millisecs";
+  private String DEFAULT_TIMEOUT_PROPERTY = "60000";
```

Comment:
To wait forever, it should be modified.

Reply:
That is what I was trying to convey earlier, the default is 60 second, which means even if user "accidentally" deletes this key, by fail-safe design Zeppelin will make sure it stays 60 seconds. However, if user wants to, can set this to "-1" which will mean infinite timeout (wait forever).

Reply:
@prabhjyotsingh I think @toughrogrammer means to make -1 by default which seems make sense. e.g. if user invoke wget in shell interpreter, it is very likely to take more than 60 seconds.

Reply:
Then wouldn't it conflict with the default recommended value that Zeppelin is shipped with http://zeppelin.apache.org/docs/0.8.0-SNAPSHOT/interpreter/shell.html#configuration. 

I mean we can make a change in all of these three places (this, doc and interpreter-setting.json), do we all agree that 60sec is too less, or should we make "-1" as the default value.

Reply:
I'm concerned about having anything waiting indefinitely...

Reply:
Based on above comment, will merge this if no more discussion.